howto/HACKME.md

```diff
@@ -148,3 +152,51 @@ dist/Debug/GNU-Linux/amos-ss17-proj4 can0
 ```
 
 To modify the build system best open the project in Netbeans (see above).
+
+## Set-Up on Virtual Machine
+If the simulator should be installed on a virtual linux machine just do the following steps to get the simulator running.
+
+ISO-TP
+
+As like in the Set-Up for the Raspberry PI, ISO-TP should be installed. If the CAN connection should be started while your system is starting, do the following steps.
+
+1. Load the can modules into the `modules.conf` file. This file can be found in your systems root directory via this path `/etc/modules-load.d`
```

Comment:
Have you tested this works as expected?
I usually use /etc/modules which should work on any Linux system whereas the modules-load.d is specific to Debian-Based systems I think.
Also if you use modules-load.d please don't call the file modules.conf but use a name that tells what it is actually for (e.g. canisotp.conf)

Reply:
Ok I will change that to the /etc/modules